crates/extension_host/src/wasm_host/wit/since_v0_6_0.rs

```diff
@@ -945,7 +945,10 @@ impl ExtensionImports for WasmState {
                                     .get(key.as_str())
                             })
                             .cloned()
-                            .context("Failed to get context server configuration")?;
```

Comment:
This change was causing the issue. This fixes the regression introduced in #32622